src/helpers/request-body.js

```
@@ -0,0 +1,52 @@
+/**
+ * Request Body Helper
+ *
+ * Serializes request bodies without treating top-level arrays as form fields.
+ */
+
+/* global module, globalThis */
```

ESLint

[error] 'globalThis' is already defined as a built-in global variable.

(no-redeclare)